src/test/scala/com/cognite/sdk/scala/v1/EventsTest.scala

```diff
@@ -73,6 +73,18 @@ class EventsTest extends SdkTest with ReadBehaviours with WritableBehaviors {
     }
   )
 
+  it should "support updating by id" in {
```

Comment:
This should also be done in a common test trait so that it's tested for all resource types.